profile/signatures.jq

```diff
@@ -0,0 +1,2 @@
+.[] |  
```

Comment:
since we call the output `.json`, might as well make it valid json. meaning, to wrap it in [ ] like in the other jq queries

Reply:
I did it at first, but it looked weird on the GitHub comment, to have all events inside an array, is harder to visualize. WDYT?

Reply:
makes sense. you could rename the file to jsonl in this case, but not a huge deal feel free to merge

Reply:
fixed!